examples/go-client/service/service-poc.go

```diff
@@ -0,0 +1,272 @@
+// +build sample_client
+
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Comcast/rulio/core"
+	"github.com/Comcast/rulio/sys"
+	"github.com/tidwall/pretty"
+	"strings"
+	"time"
+)
+
+type Service struct {
+	System  *sys.System
+	Router  interface{}
+	Stopper func(*core.Context, time.Duration) error
+}
+
+func (s *Service) SearchFact(ctx *core.Context, m map[string]interface{}) error {
```

Comment:
Why repeat all of this implementation when we have `sys.System.SearchFacts`? Can you refactor to use existing code?